data.json

```diff
@@ -874,5 +874,11 @@
     "githubUsername": "Saatvik21",
     "imageUrl": "https://user-images.githubusercontent.com/61618767/117973604-3bf81d00-b34a-11eb-86a2-8bbd7f19b6d5.png",
     "issueNumber": 493
+  },
+   {
+    "name": "Rajiv Ranjan Mars",
+    "githubUsername": "rajivranjanmars",
+    "imageUrl": "https://user-images.githubusercontent.com/67179338/118148700-7509bd80-b42e-11eb-9b52-1bd3463c4e1c.png",
+    "issueNumber": 496
   }
-]
```

Comment:
what have you done here?

Reply:
Is it good now or I need to add some more changes?

Reply:
looks good for me :)